systems/controllers/controlUtil.h

```diff
@@ -33,8 +33,9 @@ typedef struct _support_state_element
 {
   int body_idx;
   std::vector<Vector4d, aligned_allocator<Vector4d>> contact_pts;
-  int contact_surface;
   bool support_logic_map[4];
+  bool use_support_surface;
+  Vector4d support_surface;

```

Comment:
wow.  the windows compiler caught you here.  (thank you MSVC!).  

If you have a Vector4d in your class, then you need to add 
```
public:
  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```
http://eigen.tuxfamily.org/dox-devel/group__TopicStructHavingEigenMembers.html
to your class.  I should have caught that one.